tests/api/test_api_jobs.xsh

```diff
@@ -0,0 +1,22 @@
+from xonsh.api.jobs import get_last_job, get_last_pid
+from xonsh.pytest.tools import ON_WINDOWS
+import pytest
+
+
+$RAISE_SUBPROC_ERROR = True
+$XONSH_SHOW_TRACEBACK = True
+
```

Comment:
kindly use Python for tests.

Reply:
I thought Xonsh for tests would be fine because the tests for all the other `xonsh.api` modules are using Xonsh too.

Reply:
That causes some unexpected behaviours and slows down the test suit.